### DISCUSSION TITLE
Import "mesa.time" could not be resolved

### DISCUSSION THREAD

**Discussion by ahendest** [mesa/mesa#2563] [Q&A] [answered]
i can not use mesa.time. it gives me an error. Import "mesa.time" could not be resolved. İ tried this : from mesa.time import SimultaneousActivation

**Comment by quaquel** [comment, marked as answer]
Mesa.time has been deprecated in Mesa 3.0 and removed in 3.1. Please see the migration guide on read the docs for further details.
